app/controllers/admin/content/system-images/subtopics.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  actions: {
+    openModal(subtopic) {
```

Comment:
What will be it's need here then since we are creating the array in the image-modal template itself.